lib/hooks/useUppyUploader.js

```diff
@@ -0,0 +1,61 @@
+import { useState } from "react";
+
+import Uppy from "@uppy/core";
+import { useUppy } from "@uppy/react";
+import { Toastr } from "neetoui";
+
+import { UPPY_UPLOAD_CONFIG } from "components/Editor/MediaUploader/constants";
+import { convertToFileSize } from "components/Editor/MediaUploader/utils";
+import ActiveStorageUpload from "utils/ActiveStorageUpload";
+
+const useUppyUploader = ({
+  endpoint,
+  uppyConfig,
+  uppyUploadConfig = UPPY_UPLOAD_CONFIG,
```

Comment:
Since we are using the same configuration everywhere, please remove this from the props. The constant needs to be directly imported.